src/cleaning/cleaning.h

```diff
@@ -40,6 +41,7 @@ struct cleaning_policy_meta {
 
 struct ocf_cleaner {
 	void *cleaning_policy_context;
+	struct ocf_refcnt cleaning_ctx __attribute__((aligned(64)));
```

Comment:
This is wrong commit to introduce this field (it's used for the first time in the next commit), and I'd rather name it something like `refcnt` or `cleaning_refcnt`.